nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -81,6 +81,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [systemd-lock-handler](https://git.sr.ht/~whynothugo/systemd-lock-handler/), a bridge between logind D-Bus events and systemd targets. Available as [services.systemd-lock-handler.enable](#opt-services.systemd-lock-handler.enable).
 
+- [stash](https://stashapp.cc/), an organizer for your porn, written in Go. Available as [services.stash.enable](#opt-services.stash.enable).
```

Comment:
I know that this is the real description of the software, but in this case, I would definitely find an alternative here.
Maybe replace `porn` with `adult video` ?

Reply:
I think it is fine to not advertise it explicitly at all.